### DISCUSSION TITLE
Enable Kafka-UI to listen on http and https

### DISCUSSION THREAD

**Discussion by vascoferraz** [provectus/kafka-ui#3534] [Q&A] [answered]
Hi everyone :)

I am able to configure Kafka-UI to listen on http or https but not simultanesouly.

So, my question is: how can I have Kafka-UI listening on http and https simultaneously?

My idea is to redirect requests that land at `http://my-domain.com` to `https://my-domain.com`

Thanks :)

**Comment by Haarolean** [comment, marked as answer]
Hi, you have to do it with a reverse proxy.

**Reply by vascoferraz** [reply]
Thanks :)

**Reply by Haarolean** [reply]
@vascoferraz some examples:
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-traefik-proxy.yaml
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-reverse-proxy.yaml
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ssl-components.yaml

**Reply by vascoferraz** [reply]
Thanks <3
